Dockerfile

```diff
@@ -14,6 +14,7 @@
 
 FROM golang:1.13-buster AS build
 WORKDIR /go/src/github.com/vmware-tanzu/velero-plugin-for-aws
+COPY Makefile Makefile
```

Comment:
I don't think we actually need this, do we? Since we're running `go build` directly

Reply:
**What Adding Makefile Gives**
Adding the `Makefile` enables the use of
`make test-container` 
which runs the test in the container (if the container does not exist, it will build it (via target `build-test-container`).

**My two cents**
I think this is a minor addition that enables an additional `make` target.

Keep in mind it does not affect the ultimate end artifact, the container that is used at runtime (this Makefile is leftover in the build container and not transferred to the final image).

**Ultimately Your decision**
After reading all of this, if you feel I should remove the `Makefile`
please respond :-1: 
Otherwise, 
please let me keep this little file in the build container. :smile: